docs/docs/installation/manual_deploy.md

```diff
@@ -194,17 +194,17 @@ clone the project and checkout the correct branch
 if you already have the symlink environment file, you need to delete it first, clone the repo then re-run the [symlink generation command](/installation/manual_deploy#create-symlink-environment-file).
 </note>
 ```bash
-git clone git@github.com:nialldj/MyDutyPaid.git /var/www/mdc/{{timestamp_dir}}
 cd /var/www/mdc/{{timestamp_dir}}
-git fetch
-git checkout development
+git init
+git remote add origin https://github.com/nialldj/MyDutyPaid.git 
+git pull origin development
```

Comment:
Clone mean get an Identical replica,
Git init, mean, i want to track the changes of this folder. 
So you don't need to create a tracked folder and add the source then pull it...
you just clone the original...

Reply:
It will clone the master whereas we need development and staging. So we need to pull development or staging.

Reply:
that is why you fetch, and checkout the specific branch.

Reply:
Okay. What you suggest? Modify as you think and let me try that on server.

Reply:
please read this:
https://github.com/git-guides/git-init#examples-of-git-init

Reply:
I think it was correct how it was before your modifications.
did you run any issues whit my method under the deployment?